VulkanHppGenerator.cpp

```diff
@@ -5049,12 +5049,9 @@ int main(int argc, char **argv)
     template<typename ClassType> ClassType& get() VULKAN_HPP_NOEXCEPT { return static_cast<ClassType&>(*this);}
 
     template<typename ClassTypeA, typename ClassTypeB, typename ...ClassTypes>
-    std::tuple<ClassTypeA, ClassTypeB, ClassTypes...> get()
+    std::tuple<ClassTypeA&, ClassTypeB&, ClassTypes&...> get()
     {
-        return std::tuple_cat(
-            std::make_tuple(get<ClassTypeA>(),get<ClassTypeB>()),
-            std::make_tuple(get<ClassTypes>()...)
-        );
+      return std::forward_as_tuple(get<ClassTypeA>(), get<ClassTypeB>(), get<ClassTypes>()...);
```

Comment:
I think `std::tie` is more suitable for this. [Example](https://wandbox.org/permlink/UpakCDPb5DJmxvMS).